people/djcarpe.toml

```diff
@@ -0,0 +1,4 @@
+name = 'DJ Carpenter'
+github = 'djcarpe'
+github-id = 59489977
+zulip-id = 59489977
```

Comment:
Is this person's GitHub ID exactly the same as their Zulip ID?

Reply:
Yeah, that’s definitely a copy-paste mistake. Good catch!